src/app/containers/PageHandlers/__snapshots__/withContexts.test.jsx.snap

```diff
@@ -1,40 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`withContexts HOC should return all context providers 1`] = `
-<ToggleContextProvider>
-  <ServiceContextProvider
-    service="news"
-    variant={null}
-  >
-    <RequestContextProvider
-      bbcOrigin="https://www.bbc.com"
-      id="c0000000000o"
-      isAmp={true}
-      pageType="article"
-      pathname="/pathname"
-      previousPath={null}
-      service="news"
-      statusCode={200}
-      variant={null}
-    >
-      <UserContextProvider>
-        <Component
-          bbcOrigin="https://www.bbc.com"
-          data={
-            Object {
-              "status": 200,
-            }
-          }
-          id="c0000000000o"
-          isAmp={true}
-          pageType="article"
-          pathname="/pathname"
-          previousPath={null}
-          service="news"
-          variant={null}
-        />
-      </UserContextProvider>
-    </RequestContextProvider>
-  </ServiceContextProvider>
-</ToggleContextProvider>
+<div>
+  <span />
```

Comment:
I wonder if this snapshot could be better

Reply:
what is the reason for this just returning multiple `<span />`